External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -3465,26 +3476,22 @@ private static ASExpr GetExpression(ScintillaControl sci, int position, bool ign
                         sb.Insert(0, "RegExp.#" + (subCount++) + "~");
                     }
                     // array access
+                    else if (c == ']') arrCount++;
```

Comment:
This is necessary for the correct counting of parentheses, otherwise problems arise, this code already works in the ASGenerator.GetStartOfStatement In the future I want to get rid of it and keep the starting position in ASExp